packages/lodestar/src/db/api/beacon/stateCache.ts

```diff
@@ -0,0 +1,33 @@
+import {toHexString, TreeBacked, ByteVector} from "@chainsafe/ssz";
+import {BeaconState} from "@chainsafe/lodestar-types";
+
+/**
+ * Simple BeaconState in-memory cache
+ *
+ * Similar API to Repository, but synchronous
+ */
+export class StateCache {
```

Comment:
make it implement Repository it will be async and we could stub it out or provide altertive implementation easily